src/components/PositionList/index.js

```diff
@@ -146,7 +146,7 @@ function PositionList({ positions }) {
         {!below740 && <DataText area="number">{index}</DataText>}
         <DataText area="name" justifyContent="flex-start" alignItems="flex-start">
           <AutoColumn gap="8px" justify="flex-start" align="flex-start">
-            <TokenLogo size={16} address={position.pool.token.id} margin={!below740} />
+            <TokenLogo size={'16px'} address={position.pool.token.id} />
```

Comment:
why change style here?

Reply:
size unit of parameter is necessary when it type string and the source code is like this